packages/bits/demo/src/components/demo/wizard-v2/wizard-tooltip/wizard-tooltip-example.component.html

```diff
@@ -0,0 +1,41 @@
+<nui-wizard-horizontal #wizardComponent>
+
+    <!--  content of the 1st step -->
+    <nui-wizard-step-v2 i18n-label label="Very long header text. Very long header text">
+        <!-- content of the current step -->
+        <span i18n>Step with long header text</span>.
+
+        <!-- content of the footer -->
+        <ng-template wizardStepFooter>
+            <ng-container *ngTemplateOutlet="wizardNext"></ng-container>
+        </ng-template>
+    </nui-wizard-step-v2>
+
+    <!--  Step with custom label template -->
+    <nui-wizard-step-v2 i18n-label tooltip="Custom Label Template">
+        <ng-template wizardStepLabel>
```

Comment:
Similar to previous comment, do you mind creating a ticket for renaming this selector to `nuiWizardStepLabel`? Alternately, doing it in this PR is fine as well.

Reply:
done